app/templates/account/settings.hbs

```diff
@@ -76,4 +76,91 @@
       {{loading-indicator}}
     {{/if}}
   </section>
+  {{#if features.proVersion}}
+    <section class="active-visiblity-settings" data-test-active-visiblity-settings>
+      <h3>Insights Visibility Settings</h3>
+      <div class="insights-settings">
+        {{!-- <div class="name" data-test-insights-settings-title>Insights</div> --}}
+        <p class="description" data-test-insights-settings-description>
+          Make more informed decisions about your development workflow using your build Insights.<br />
+          {{#link-to 'owner.repositories' account.login (query-params tab='insights')}}View {{account.name}}'s Insights{{/link-to}}
```

Comment:
Further to my commentary above, this kind of string-construction is something I’d like to see an assertion of.